pkg/controllers/binding/common.go

```diff
@@ -302,8 +302,8 @@ func needReviseReplicas(replicas int32, placement *policyv1alpha1.Placement) boo
 	return replicas > 0 && placement != nil && placement.ReplicaSchedulingType() == policyv1alpha1.ReplicaSchedulingTypeDivided
 }
 
-func shouldSuspendDispatching(suspension *policyv1alpha1.Suspension, targetCluster workv1alpha2.TargetCluster) bool {
-	if suspension == nil {
+func shouldSuspendDispatching(suspension *workv1alpha2.Suspension, targetCluster workv1alpha2.TargetCluster) bool {
+	if suspension == nil || suspension.Suspension == nil {
```

Comment:
Why?

Reply:
This statement `suspendDispatching := ptr.Deref(suspension.Dispatching, false)` below will refer `suspension.Dispatching` and it would panic if suspension.Suspension ==nil.